### PR TITLE
Simplifies the readme

### PR DESCRIPTION
**Pull request by xgerman**

### CODE REVIEW

**Review by johnsom — CHANGES_REQUESTED** [review]

**Comment by xgerman** [comment]
recheck_xenial_functional_test

**Comment by xgerman** [comment]
recheck_xenial_functional_test

**Comment by xgerman** [comment]
recheck_xenial_functional_test

**Comment by xgerman** [comment]
recheck_xenial_functional_test

File: README.md [hunk 1]
```diff
@@ -1,80 +1,10 @@
...
-    ./gating/[pre_merge_test|post_merge_test]/[pre|run|post]
-
-4. Once you ran pre and it failed you might need to run:
-
-    apt-get install libicu55=55.1-7ubuntu0.2  libssl1.0.0=1.0.2g-1ubuntu4.8
-
-
+* Branch master is for the managed k8s install against Newton
+* Branch pike and pike-rc are for Pike
+* For Wueens and beyond this repository is retired and installs will be using the upstream OSA os_octavia
```
Comment (johnsom) [inline README.md]: Wueens -> Queens

**Review by johnsom — CHANGES_REQUESTED** [review]

**Review by johnsom — APPROVED** [review]

**Pull request merged**